src/image_modifications.cpp

```diff
@@ -618,7 +617,7 @@ REGISTER_MOD_PARSER(TC, args)
 		return nullptr;
 	}
 
-	const int side_n = lexical_cast_default<int>(params[0], -1);
+	const int side_n = utils::from_chars<int>(params[0]).value_or(-1);
```

Comment:
This doesn't seem to be your fault, but… the error message 2 lines down is wrong, maybe you could fix that while you're at it? It should be outputting `params[0]`, not `side_n`.

Reply:
Wil do.